src/cardboard-vr-display.js

```diff
@@ -52,9 +52,10 @@ function CardboardVRDisplay() {
   // Set the correct initial viewer.
   this.deviceInfo_.setViewer(this.viewerSelector_.getCurrentViewer());
 
-  this.rotateInstructions_ = new RotateInstructions();
-
-  this.injectPresentModeCssClass_();
+  if (!WebVRConfig.ROTATE_INSTRUCTIONS_DISABLED) {
+    this.rotateInstructions_ = new RotateInstructions();
+    this.injectPresentModeCssClass_();
```

Comment:
This should happen regardless of whether or not instructions are disabled (ie. out of the if statement). It's a fix for fullscreen mode.